tests/CronTaskControllerTest.php

```diff
@@ -112,7 +112,7 @@ public function testRunTask()
     public function testDefaultQuietFlagOutput()
     {
         $this->loginWithPermission('ADMIN');
-        $this->expectOutputRegex('#'.SS_Datetime::now()->Format('Y-m-d').'#');
+        $this->expectOutputRegex('#' . DBDatetime::now()->Format('Y-m-d') . '#');
```

Comment:
Ah, we also changes the date format =/ lets see if the tests pass but we'll probably need to update this too